data/accountManager.go

```diff
@@ -31,23 +30,11 @@ import (
 	"github.com/algorand/go-algorand/protocol"
 )
 
-// A ParticipationKeyIdentity defines the parameters that makes a pariticpation key unique.
-type ParticipationKeyIdentity struct {
```

Comment:
This is a little different than my ParticipationID, but they both seem to do the same thing so I'm merging them.

Reply:
I think that it's ok - but you'll also need `VoteID:      participation.Voting.OneTimeSignatureVerifier` so that it would capture both.